### DISCUSSION TITLE
Template two - no semantic marking of the navigation in the header

### DISCUSSION THREAD

**Discussion by WM-Loose** [templaza/astroid-framework#688] [General]
In template two, there is actually no semantic marking of the navigation in the header. It is correct that the menu in the header must be marked as <NAV in order to give screen reader users an indication of the type of content.

Instead, there are two <nav in the footer, but they have no ID and no labels.
And other errors regarding accessibility.

**Comment by Cvpher** [comment]
Have you found out anything about this topic yet?
Greetings!
